### DISCUSSION TITLE
Valheim Server Web Interface

### DISCUSSION THREAD

**Discussion by Peabo83** [Nimdy/Dedicated_Valheim_Server_Script#229] [Ideas]
Hey Nimdy! Great script, what do you think about building a web GUI? I threw together a working example:

https://github.com/Peabo83/Valheim-Server-Web-GUI

I haven't worked in much functionality yet, but if you think this is worth pursuing I'll keep working on it.

**Comment by Peabo83** [comment]
To the one person who up-doot'ed this, I built it out.

https://github.com/Peabo83/Valheim-Server-Web-GUI

**Reply by Nimdy** [reply]
nice job!

:)  I will take a later at it later for sure!

**Reply by Nimdy** [reply]
@Peabo83 do you want to bring this into the Njord Menu?

We can all work on it together under one project.  

I was going to test this later and then build a complete auto installer for everything.

**Reply by Peabo83** [reply]
Sounds like a plan to me! I sent you a PM on discord.
